### DISCUSSION TITLE
Domotizzare radiocomando rolling code

### DISCUSSION THREAD

**Discussion by Dario2502** [indomus/forum#1744] [GENERALE - Discussioni generiche]
Ciao a tutti,
sono nuovo del forum e sono in cerca del vostro aiuto.

Ho 3 tende da sole che voglio domotizzare che sono comandate unicamente da radiocomandi rolling code.
Leggendo sul sito, apparentemente l'unica soluzione sarebbe quella di usare un attuatore a contatto pulito.
Visto che i radiocomandi sono 3 ed ognuno ha 3 tasti (salita, discesa e stop), oltre ovviamente a dover comprare 3 radiocomandi, devo comprare 9 attuatori per poter controllare tutto da remoto?
Non c'è una soluzione un po' più smart?
Grazie a tutti per l'aiuto

**Comment by Buff81** [comment]
Sono tende Somfy? Io ho risolto installando un trasmettitore a contatti puliti Somfy art.1810135. Si associa alla tenda pari pari ad ad un telecomando, e puoi collegarci dei pulsanti e/o dei relé a contatto pulito. Io l'ho installato con due Shelly 1 ed ho guadagnato i pulsanti fisici e la possibilità di comandare la tenda in modo smart (il comando "stop" lo si ottiene con l'azione simultanea di entrambi i contatti). 
Il suo limite è che devi avere spazio in una scatola di derivazione per installarlo, o almeno la possibilità di montare una scatola da esterno extra. In più sommando il costo del trasmettitore, 2 shelly, pulsanti, cavi, canaline e scatole varie non mi è venuto  poco come spesa. E ti toccherebbe una spesa X3... In ogni caso è una soluzione più pulita che aprire telecomandi e saldare sui contatti, per poi "nascondere" da qualche parte (visto che devi alimentare i relé in qualche modo).

**Comment by Dario2502** [comment]
Purtroppo no. sono tende con motore Nice TTGO.
Dalle descrizioni di questo articolo non leggo che supporta il rolling code quindi non credo sia compatibile con le mie riceventi.

**Comment by Buff81** [comment]
Quell'articolo è specifico per tende Somfy, non credo sia compatibile con altre marche (anche se ti assicuro che i telecomandi sono rolling code). Mi spiace, ho sbattuto la testa un po' per scoprire come fare con la mia (che ho trovato installata quando ho comprato casa). Se ti può aiutare, ho trovato la soluzione cercando come montare pulsanti fisici per comandare la tenda, prova quella strada.

**Comment by RobyBS** [comment]
mi mettente il link per cortesia .. 👍 👍 👍 
sono curioso ...

**Reply by Buff81** [reply]
Parlavi con me?

https://www.somfy.it/soluzioni/1810135/trasmettitore-radio-rts

https://asset.somfy.com/Document/a1833274-0f49-4ca3-93ac-862070e2223e_1810135_descrizione_collegamento_con_domotica.pdf

ho messo 2 shelly 1 in parallelo ai pulsanti, impostati per impulso da 0.5 s: funziona.

(N.B. il trasmettitore l'ho pagato la metà di quanto indicato sul sito Somfy: 165 € è un furto a mano armata, 80 un furto e basta).

**Reply by RobyBS** [reply]
GRAZIE ! .. 👍

**Reply by RobyBS** [reply]
scusa se torno sull'argomento ... ma se invece di mettere due shelly1 , utilizzi uno shelly 2.5 ... non riusciresti ad "intercettare" anche la posizione ( o percentuale ) di apertura della tenda ? ...

**Comment by Dario2502** [comment]
Avevo già pensato di montare un pulsante ma purtroppo è comandabile solo da telecomando. Non hanno la predisposizione per i comandi fisici.
